internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -355,6 +377,13 @@ func resourceLogAnalyticsWorkspaceRead(d *pluginsdk.ResourceData, meta interface
 				d.Set("daily_quota_gb", utils.Float(-1))
 			}
 
+			if features := props.Features; features != nil {
+				v, ok := (*features).(map[string]interface{})["allow_resource_only_permissions"]
+				if ok {
+					d.Set("allow_resource_only_permissions", v)
+				}
+			}
```

Comment:
This needs a value set into state, which would be solved by setting the Default value in the schema above
```suggestion
                        allowResourceOnlyPermissions := false
			if features := props.Features; features != nil {
				v, ok := (*features).(map[string]interface{})["allow_resource_only_permissions"]
				if ok {
					allowResourceOnlyPermissions = *v
				}
			}
			d.Set("allow_resource_only_permissions", allowResourceOnlyPermissions)
```